__tests__/main.test.ts

```diff
@@ -86,3 +86,46 @@ describe('When the script fails to run', () => {
     expect(fakeSetFailed).toBeCalledWith('the error message');
   });
 });
+
+describe('When running the action with the cake-bootstrap input argument', () => {
+  const fakeGetInput = core.getInput as jest.MockedFunction<typeof core.getInput>;
+  const fakeCakeTool = CakeTool as jest.MockedClass<typeof CakeTool>;
+
+  test('it should bootstrap the default Cake script', async () => {
+    fakeGetInput.mockImplementation(
+      key => {
+        switch (key) {
+          case 'cake-bootstrap':
+            return 'true';
+          default:
+            return '';
+        }
+      }
+    );
+    await run();
+    expect(fakeGetInput).toBeCalledWith('cake-bootstrap');
+    expect(fakeCakeTool.bootstrapScript).toBeCalledWith(
```

Comment:
If you don't care about the arguments of the call, you can simply say `toBeCalled`:

```suggestion
    expect(fakeCakeTool.bootstrapScript).toBeCalled();
```